cli/command/service/list.go

```diff
@@ -116,7 +118,7 @@ func printTable(out io.Writer, services []swarm.Service, running map[string]int)
 		if service.Spec.Mode.Replicated != nil && service.Spec.Mode.Replicated.Replicas != nil {
 			replicas = fmt.Sprintf("%d/%d", running[service.ID], *service.Spec.Mode.Replicated.Replicas)
 		} else if service.Spec.Mode.Global != nil {
-			replicas = "global"
+			replicas = fmt.Sprintf("%d/%d global", running[service.ID], tasksPerService[service.ID])
```

Comment:
No, actually it is not the total node count. Here is scene:
1. 5 nodes, 2 managers, 3 workers.
2. docker service create --constraint node.role==manager ubuntu:14.04 sleep 100000
3. before 1.13.0,  global service will be every node even if the node does not meet the constraint. While since PR https://github.com/docker/swarmkit/pull/1570, task will only be on nodes which satisfies the constraints. However this PR counts all tasks of a global service, and I think it will be OK for docker version both before 1.13- and 1.13+